tools/compare-perf/performance.js

```diff
@@ -368,7 +368,7 @@ async function runPerformanceTests( branches, options ) {
 			const resultsRounds = resultFiles
 				.filter( ( file ) =>
 					file.includes(
-						`${ testSuite }_${ sanitizedBranchName }_round-`
+						`/${ testSuite }_${ sanitizedBranchName }_round-`
```

Comment:
Since we have two test suites, one for the `editor` and another one for the `product editor`, we added a slash "/" here because `editor` was included in `editor_trunk_round-` and in `product-editor_trunk_round-`.